Ontology/Asset/Component/HVACCoolingMethod.json

```diff
@@ -0,0 +1,78 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:asset:HVACCoolingMethod;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Cooling Method"
+  },
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "type",
+      "displayName": {
+        "en": "Type"
+      },
+      "writable": true,
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+          {
+            "name": "DX",
+            "displayName": {
+              "en": "DX"
+            },
+            "enumValue": "DX"
+          },
+          {
+            "name": "ChilledWater",
+            "displayName": {
+              "en": "Chilled Water"
+            },
+            "enumValue": "chilledWater"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Property",
+      "name": "rows",
+      "displayName": {
+        "en": "Rows"
+      },
+      "writable": true,
+      "schema": "integer",
+      "comment": "Coil row configuration"
+    },
+    {
+      "@type": "Property",
+      "name": "finsPerInch",
+      "displayName": {
+        "en": "Fins per Inch"
+      },
+      "writable": true,
+      "schema": "integer",
+      "comment": "Fins per inch"
+    },
+    {
+      "@type": "Property",
+      "name": "outsideDiameter",
+      "displayName": {
+        "en": "Outside Diameter"
+      },
+      "writable": true,
+      "schema": "double",
+      "comment": "inch"
```

Comment:
Imperial-specific comment, will not be included.

Reply:
Agree.